docs/workforce-management-framework/upstream-1/working-with-keycloak.adoc

```diff
@@ -1,28 +1,18 @@
 include::topics/shared/attributes.adoc[]
 
-//':chapter:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-:chapter: working-with-keycloak
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: working-with-keycloak
 
 = Working With {WFM-RC-KeycloakName}
 
 //Concepts
-The {WFM-RC-NameShort} Demo Application allows for authentication and access management using link:http://www.keycloak.org/index.html[{WFM-RC-KeycloakName}]
-however, {WFM-RC-KeycloakName} is not the default authorization and authentication module for {WFM-RC-NameShort}.
-{WFM-RC-KeycloakName} integration is configured within the Demo Application.
-
-NOTE: To configure your own instance of {WFM-RC-KeycloakName}, see the link:{WFM-RC-KeycloakURL}getting_started/topics/first-boot.html[{WFM-RC-KeycloakName} Getting Started Guide].
+include::topics/con-keycloak.adoc[leveloffset=+1]
 
 //Procedures
 include::topics/pro-keycloak-implementation.adoc[leveloffset=+1]
 include::topics/pro-keycloak-enablement.adoc[leveloffset=+1]
+For login details, refer to the xref:{context}-credentials-for-the-demo-application[Credentials for the {WFM-RC-KeycloakName} Admin Console] section of xref:{context}-ref-demo-app[{WFM-RC-KeycloakName} Reference Material].
 
-== {WFM-RC-KeycloakName} Reference Material
```

Comment:
this content is now part of "include::topics/ref-keycloak.adoc[leveloffset=+1]" (see new change -->) and imported